examples/network/expressroutecircuits.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: network.azure.crossplane.io/v1alpha3
+kind: ExpressRouteCircuits
+metadata:
+  name: example-vn
+spec:
+  resourceGroupName: test-exp
+  location: West US 2
+  properties:
+    allowClassicOperations: False
+    bandwidthInMbps: 50
+    globalReachEnabled: True
+    peeringLocation: Chennai2
+    serviceProviderName: Airtel
+  sku:
+    family: MeteredData
+    tier: Standard  
+  providerConfigRef:
+    name: default
```

```suggestion
    name: example
```
In the example manifests, for references to other resources for which an example manifest exists, the name in that manifest is used, and the [example provider config](https://github.com/crossplane/provider-azure/blob/33f728dbaadf9c9448d07ea16119d3137023d34d/examples/azure-provider.yaml#L16) name is `example`. This makes the life a bit easier when trying things out, and for conformance tests.